src/helpers/discord_utils.ts

```diff
@@ -258,14 +258,15 @@ export async function sendOptionsMessage(messageContext: MessageContext, guildPr
         {
             title: updatedOption === null ? "Options" : `${updatedOption.option} ${updatedOption.reset ? "reset" : "updated"}`,
             description:
-                `Now playing the ${optionStrings[GameOption.LIMIT]} most popular songs out of the __${totalSongs.countBeforeLimit}__ by ${guildPreference.isGroupsMode() ? `${optionStrings[GameOption.GROUPS]} (${optionStrings[GameOption.SUBUNIT_PREFERENCE]})` : `${optionStrings[GameOption.GENDER]} ${optionStrings[GameOption.ARTIST_TYPE]}`}\
+                `-Now playing the ${optionStrings[GameOption.LIMIT]} most popular songs out of __${totalSongs.countBeforeLimit}__ by ${guildPreference.isGroupsMode() ? `${optionStrings[GameOption.GROUPS]} (${optionStrings[GameOption.SUBUNIT_PREFERENCE]})` : `${optionStrings[GameOption.GENDER]} ${optionStrings[GameOption.ARTIST_TYPE]}`}\
```

Comment:
We should get rid of all the sentences. To simplify all of this, you should change the format to something like:
```
Limit: 1600
Gender: Male/female
Years: 2015-2018
Language: Korean-only
OSTs: Exclude
```
with all the options

Reply:
Here's something to get you started. Each entry would look something like this (don't follow exactly, I just came up with them on the spot):
```
LIMIT => 1600 - 3200 / ${totalSongs.countBeforeLimit}
GROUPS => Groups: Red Velvet, IU, Heize
SUBUNIT_PREFERENCE => Subunits: Included
GENDER => Gender: Male/Female
CUTOFF => Years: 2015-2018
ARTIST_TYPE => Artist Type: Soloist
EXCLUDE => Excluding: LOONA
INCLUDE => Including: Blackpink
SEEK_TYPE => Seek: Beginning
OST_PREFERENCE => OST songs: Include
RELEASE_TYPE => Release: Official
MODE_TYPE => Guess mode: Song
GOAL => Goal: 50
TIMER => Timer: 3 seconds
SHUFFLE_TYPE => Shuffle: Unique
RELEASE_TYPE => All songs including dance practices, acoustic versions, remixes, etc.
MULTIGUESS => Multiguess: On
```
Follow the same logic that decides what to show.

Reply:
Alright , would make it this way

Reply:
Shall we keep the codeLine effect ?
`optionStrings[gameOption] = (updatedOption && updatedOption.option) === gameOption ? bold(gameOptionString) : codeLine(gameOptionString);`

Reply:
I think we should

Reply:
ok